### ISSUE TITLE
Table 'performance_schema.session_variables' doesn't exist

### ISSUE THREAD

**Issue by AyushyaChitransh** [MartinThoma/flask_mysql_dockerized#1]
From the question [here](https://stackoverflow.com/q/45247634/6207775) the present possible fix for this issue is:

 ```
Start the containers:
  # docker-comopse up

Find the container ID of the mysql image:
  # docker container ls

Enter the container
  # sudo docker exec -it 4f7d3f0763ad bash

Upgrade
  # mysql_upgrade -u root -p --force

Exit the container, then restart
  # docker-compose up

Initialize the database:
  # cd /docker-entrypoint-initdb.d/
  # mysql -h localhost -u root -p
     mysql> source flask_db.sql;
```

---

I was not able to build the dockerfile due to low internet resources availability at my place. But I would really like to know if there were any fruitful results by adding:

```
volumes:
  - ./flask_db.sql:/docker-entrypoint-initdb.d/flask_db.sql
```

to the `db` service. Also, I doubt that if there is a possibility of mysql getting late in starting up and it may be possible that `web` is trying to access `db` before `db` could initialize the database and users? In that case we might need a `wait-for` script.

**Comment by MartinThoma** [comment]
Yes, your answer helped (I upvoted it). This is, however, still not the kind of solution I was looking for.

I don't understand why you post the fix I wrote here.

**Comment by AyushyaChitransh** [comment]
Because I was unable to replicate the issue on my system. Mainly because I was unable to install the requirements in the dockerfile. And I wanted to discuss some more things regarding the dockerfile, which might not have been relevant to the question there. 

Or should I change the subject of this issue to a better one. One which I originally intended for: "Finding a base image"
